src/Sign.Core/DataFormatSigners/DynamicsBusinessCentralAppFileType.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE.txt file in the project root for more information.
+
+namespace Sign.Core
+{
+    internal sealed class DynamicsBusinessCentralAppFileType : ISignableFileType
+    {
+        private const string FileExtension = ".app";
+
+        private readonly ReadOnlyMemory<byte> _expectedHeader;
+
+        internal DynamicsBusinessCentralAppFileType()
+        {
+            _expectedHeader = new byte[] { 0x4e, 0x41, 0x56, 0x58 }; // NAVX
+        }
+
+        public bool IsMatch(FileInfo file)
```

Comment:
How would any I/O or file access error manifest in the tool?  Not sure if this method should handle those and bubble up a friendly message, if the caller would handle that, or if the tool would just crash.

Reply:
This PR does not change how I/O errors are reported.  Both before and after this change, the tool would fail with a message like "Could not find file '___'." and a callstack and a failing exit code.

There is room for fine tuning here, either dealing with files which are inaccessible (say, due to permissions) or in how we report the errors, but we haven't received user feedback in this area yet.